test/library/draft/DistributedMap/v3/ChainTable.chpl

```diff
@@ -0,0 +1,378 @@
+
+// A fork of ChapelHashtable that uses chaining to resolve collisions instead of linear probing
+
+module ChainTable {
+
+    use ChapelBase, DSIUtil;
+    private use CTypes;
+
+    const defaultInitialCapacity : uint = 32;
+    param numLocalBucketSlots : uint = 2;
+
+    const loadFactorGrowThreshold : real = 1.5,
+          loadFactorShrinkThreshold: real = 0.125;
+
+    // ----------------------------------------------------
+    // Component data structures for Bucket implementation
+    // ----------------------------------------------------
+
+    // empty needs to be 0 so memset 0 sets it
```

Comment:
I know you inherited this, but let's clarify that this refers to later code by naming the function that handles it